tools/common/common-build.targets

```diff
@@ -21,12 +21,8 @@
       <_ImportLibraryFileNames Include="@(Link -> '%(ImportLibrary)')">
       </_ImportLibraryFileNames>
 
-      <_IWBuiltProjectOutputGroupOutput Include="@(BuiltProjectOutputGroupOutput -> '%(FinalOutputPath)')" />
```

Comment:
wait, now i'm really concerned... wasn't this, like, the _thing_ that made things work?

Reply:
As I understand it, the new lines replace what this was doing. This keeps the full path but also ensures distinction.